{"name": "我是cate的第四条用例!", "status": "failed", "statusDetails": {"message": "Failed", "trace": "@allure.title(\"我是cate的第四条用例!\")\n    def test_04():\n        print(\"我是cate的第四条用例!\")\n>       pytest.fail()\nE       Failed\n\nTestcase_Api\\Shop\\test_cate.py:22: Failed"}, "attachments": [{"name": "log", "source": "5a072005-e9b3-46b7-a5f8-c778410eb4fe-attachment.txt", "type": "text/plain"}], "start": 1670495042662, "stop": 1670495042662, "uuid": "bfc076c7-fd89-4263-bb92-eb8bb8210e8e", "historyId": "ec3cd6a6ccc41a92ae90897b1d080f2a", "testCaseId": "a1d8ad9c228ca0fd62f8507d21f798b9", "fullName": "Testcase_Api.Shop.test_cate#test_04", "labels": [{"name": "parentSuite", "value": "Testcase_Api.Shop"}, {"name": "suite", "value": "test_cate"}, {"name": "host", "value": "LAPTOP-1IKMED1I"}, {"name": "thread", "value": "18064-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcase_Api.Shop.test_cate"}]}